cypress/e2e/scenario-2.1-basic-service.cy.js

```diff
@@ -0,0 +1,243 @@
+    // Check if only one row has been added to the table.
+    cy.get('[aria-label="InstanceRow-Intro"]').should("have.length", 1);
+  });
+
+  it("2.1.3 Edit previously created instance", () => {
+    cy.visit("/console/");
+    cy.get('[aria-label="Environment card"]').contains("lsm-frontend").click();
+    // Expect to find one badge on the basic-service row.
+    cy.get("#basic-service")
+      .get('[aria-label="Number of instances by label"]')
+      .children()
+      .should("have.length", 1);
+    cy.get("#basic-service").contains("Show inventory").click();
+    cy.get("#expand-toggle0").click();
+
+    // expect row to be expanded
+    cy.get(".pf-c-table__expandable-row-content").should("to.be.visible");
+
+    // Expect to find status tab
+    cy.get(".pf-c-tabs__list li:first").should("have.class", "pf-m-current");
+
+    // Expect edit button to be disabled
+    cy.get(".pf-c-description-list").contains("Edit").should("be.disabled");
+    cy.get('[aria-label="LegendItem-lone"]').should(
+      "contain",
+      "No resources available yet"
+    );
+
+    // await compilation to check state
+    waitForCompile();
+    cy.wait(2000);
+
+    // check state is up now
+    cy.get('[aria-label="InstanceRow-Intro"]:first')
+      .find('[data-label="State"]')
+      .should("contain", "up");
+
+    // click on edit button
+    cy.get(".pf-c-description-list").contains("Edit").click();
+
+    // check if the amount of fields is 4 instead of 11
+    cy.get("form").find("input").should("have.length", 4);
+
+    // delete first value and submit should give an error toast
+    cy.get("#address_r1").clear();
+    cy.get("button").contains("Confirm").click();
+    cy.get(".pf-m-danger").should("be.visible");
+
+    // Fill in first value 1.2.3.8/32 and submit
+    cy.get("#address_r1").type("1.2.3.8/32");
+    cy.get("button").contains("Confirm").click();
+
+    // expect to land on Service Inventory page and to find attributes tab button
+    cy.get(".pf-c-tabs__list").contains("Attributes").click();
+
+    // Expect to find new value as candidate and old value in active and no rollback value
+    cy.get('[aria-label="Row-address_r1"')
+      .find('[data-label="candidate"]')
+      .should("contain", "1.2.3.8/32");
+    cy.get('[aria-label="Row-address_r1"')
+      .find('[data-label="active"]')
+      .should("contain", "1.2.3.5/32");
+    cy.get('[aria-label="Row-address_r1"')
+      .find('[data-label="rollback"]')
+      .should("contain", "");
+  });
+
+  it("2.1.4 Delete previously created instance", () => {
+    cy.visit("/console/");
+
+    // Add interceptios for the delete and get call to be able to catch responses later on.
+    cy.intercept("DELETE", "/lsm/v1/service_inventory/basic-service/**").as(
+      "DeleteInstance"
+    );
+    cy.intercept(
+      "GET",
+      "/lsm/v1/service_inventory/basic-service?include_deployment_progress=True&limit=20&&sort=created_at.desc"
+    ).as("GetServiceInventory");
```

Comment:
If I am not mistaken, as you don't use it @GetServiceInventory further in that test case, there is no need to "intercept.as" just intercept

Reply:
You're right, I had forgotten to remove it :D

Reply:
this is now fixed :)